### ISSUE TITLE
CodeErroe

### ISSUE THREAD

**Issue by iszhihao** [Xharlie/pointnerf#101]
In this code, are `points_color_all` and `points_dir_all` reversed when using `torch.cat`?

if points_xyz_all is not None:
            if opt.bgmodel.startswith("planepoints"):
                gen_pnts, gen_embedding, gen_dir, gen_color, gen_conf = train_dataset.get_plane_param_points()
                visualizer.save_neural_points("pl", gen_pnts, gen_color, None, save_ref=False)
                print("vis pl")
                points_xyz_all = torch.cat([points_xyz_all, gen_pnts], dim=0)
                points_embedding_all = torch.cat([points_embedding_all, gen_embedding], dim=1)
                points_color_all = torch.cat([points_color_all, gen_dir], dim=1)
                points_dir_all = torch.cat([points_dir_all, gen_color], dim=1)
                points_conf_all = torch.cat([points_conf_all, gen_conf], dim=1)
            model.set_points(points_xyz_all.cuda(), points_embedding_all.cuda(), points_color=points_color_all.cuda(),
                             points_dir=points_dir_all.cuda(), points_conf=points_conf_all.cuda(),
                             Rw2c=normRw2c.cuda() if opt.load_points < 1 and opt.normview != 3 else None)
            epoch_count = 1
            total_steps = 0
            del points_xyz_all, points_embedding_all, points_color_all, points_dir_all, points_conf_all
